best-practices/native/nativescript/images.md

```diff
@@ -0,0 +1,72 @@
+# Handle image in Nativescript-Angular
+
+To handle image in Nativescript-Angular sometimes looks hard and frustrating to do so.
+
+Doing some research I found this video from Alexander Ziskind:
+
+[Use a Hook to Process Resource Images | NativeScript Tutorial by Alexander Ziskind](https://www.youtube.com/watch?v=KgK_d-1lG84&t=46s)
+
+That video shows how `nativescript-images-generator-hook` plugin can make that task very easy.
+
+[Images Generator Hook | NativeScript Marketplace ](https://market.nativescript.org/plugins/nativescript-images-generator-hook)
+
+This plugin runs only in build moment. It is a build hook called to verify all images added in `App_resources\images` folder. Then its create copy from each image in small scale and save in to the respective folder,its depends witch platform you are building the app (android/ios).
+
+### How to setup:
+
+1 - Install the plugin
+`npm install nativescript-images-generator-hook --save-dev`
+
+2 - Create a folder named `images` inside `App_Resources` folder.
+3 - Add all our images in `images` folder.
```

Comment:
```suggestion
3 - Add all of our images in the `images` folder.
```

I don't know the grammar rule so I had to look it up: https://english.stackexchange.com/questions/65650/all-our-x-vs-all-of-our-x

Didn't know stackexchange had an English portion!